ArduCopter/Copter.cpp

```diff
@@ -260,7 +260,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if HAL_BUTTON_ENABLED
     SCHED_TASK_CLASS(AP_Button,            &copter.button,              update,           5, 100, 168),
 #endif
-#if AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED
+#if AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED && AP_INERTIALSENSOR_HARMONICNOTCH_ENABLED
```

Comment:
Not necessary, this is already covered by the define of AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED